frontend/src/pages/projects/screens/projects/EmptyProjects.tsx

```diff
@@ -27,6 +27,7 @@ const EmptyProjects: React.FC = () => {
       <EmptyStateSecondaryActions>
         <Button
           href="/notebookController"
+          component="a"
```

Comment:
So every button dedicated to navigation should have the component prop to a hyperlink right?

Reply:
unless there is a `<Link ... />` component inside the button. In that case i dont think you need component='a' 

@jenny-s51 please confirm if this is the case

Reply:
You are correct @Gkrumbach07 - reverted in https://github.com/opendatahub-io/odh-dashboard/pull/983/commits/fbad4010e0eefada39d8a983144c99babcfcd033